index.html

```diff
@@ -9,7 +9,7 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
     <link rel="stylesheet" href="css/reveal.css">
-    <link rel="stylesheet" href="palantirnet-theme/assets/css/styles.css" id="theme">
+    <link rel="stylesheet" href="palantir-reveal/palantirnet-theme/assets/css/styles.css" id="theme">
     <!-- Printing and PDF exports -->
```

Comment:
this makes sense if you want to drop the whole dir right into your reveal, I wrote it the way I did because really you should take the stuff out of "palantir-reveal" dir, and place it into your new dir.

I'm open to either way, but I think we should update the readme to reflect whichever option we go w/ so that the paths can be correct. (currently it is reflecting taking contents out of palantir-reveal, and placing that in your reveal dir)